### DISCUSSION TITLE
Detecting corner points problem

### DISCUSSION THREAD

**Discussion by Melynt3** [capawesome-team/capacitor-mlkit#122] [General]
Yo, im getting weird corner points logs, I logged it and it feels random :D Im using demo version of Robin as base, to detect and scale those corners in device, html is almost identical.

logs:

```xml
{"rawValue":"ASDSADASDAS","bytes":[71,80,76,50,50,48,53,48,53,50,56,50,57],"cornerPoints":[[686,973],[681,1069],[165,1094],[174,1008]],"format":"CODE_128","valueType":"TEXT","displayValue":"ASDSADASDAS"}
⚡️  [log] - Detection Corner Points: [[57,519.7821655273438],[357,519.7821655273438],[357,644.7821655273438],[57,644.7821655273438]]
⚡️  [log] - Current Corner Points: [[686,973],[681,1069],[165,1094],[174,1008]]
⚡️  [log] - Conditions not met. Returning...
```

It makes problem because as u can see in logs scan never getting the value and enter never ending loop, when im commenting corner detection scanner works fine. 

Anyone occured same error, or know what can i do in this situation.

**Comment by Melynt3** [comment]
The corner point problem occurs only on iOS device in my case. 

i just can't understand why iOS coords are different on every angle im attempting to scan. Is there a way to lock it somehow? 

Would be greatfull for some tips/help

**Reply by robingenz** [reply]
This is not enough information to be able to help you. Please have a look at the [official demo app](https://github.com/robingenz/capacitor-mlkit-plugin-demo).

**Reply by Melynt3** [reply]
To reproduce it u have to run ur demo app on iOS and log current corner points and corner point detected to see that its scaling not too good. 

I tired ur demo b4, and even adapted this part where it catches the box but works only on android for me.

**Reply by Melynt3** [reply]
i even saw the #115  issue is similar and i think that might be the case, bacause the coords are different from different angles

**Reply by robingenz** [reply]
Please create a screen recording and show the problem there.

**Comment by robingenz** [comment]
For ref https://github.com/capawesome-team/capacitor-mlkit/issues/126
